services/blockchain-connector/shared/sdk/client.js

```diff
@@ -62,7 +62,7 @@ const instantiateClient = async (isForceUpdate = false) => {
 			return clientCache;
 		}
 
-		if ((Date.now() - instantiationBeginTime) > MAX_INSTANTIATION_WAIT_TIME) {
+		if (Date.now() - instantiationBeginTime > MAX_INSTANTIATION_WAIT_TIME) {
```

Comment:
Wont brackets offer better readability?
```suggestion
		if ((Date.now() - instantiationBeginTime) > MAX_INSTANTIATION_WAIT_TIME) {
```